.gitignore

```diff
@@ -31,10 +31,14 @@ yarn-error.log*
 # vercel
 .vercel
 
+#npm
+package-lock.json
+
```

Comment:
The `package-lock.json` file is meant to be committed to the repo and not ignored. That way all of us can install exactly the same dependencies every time and avoid "works on my machine" bugs.

https://docs.npmjs.com/cli/v8/configuring-npm/package-lock-json#:~:text=This%20file%20is%20intended%20to%20be%20committed%20into%20source%20repositories

Reply:
noted. 
will make modifications needed